components/new-contribution-flow/index.js

```diff
@@ -217,6 +221,32 @@ class ContributionFlow extends React.Component {
     return encodeURIComponent(currentPath);
   }
 
+  /** Validate step profile, create new incognito/org if necessary */
+  /** TODO: create profile for new org */
```

Comment:
I don't understand what this todo is about?

Reply:
Do we not want the 'create new org' form on the profile section like with the current contribution flow? If so we have to add it to the `validateStepProfile` like in the old flow so we create the new org in the db before moving on to the payment step, so it has an id to put into the order mutation.

Reply:
As per Slack discussion, will add in the designs Memo has put together for new org but will do that in a separate PR